### PR TITLE
Add assets documentation

### PR DESCRIPTION
**Pull request by SirPryderi**
Add documentation on the assets tool and CAD conversions.

### CODE REVIEW

File: FORMAT.md [hunk 1]
```diff
@@ -35,13 +36,14 @@ westport-house.zip/
 ...
 ```
 
-An example of this file structure can be found in the [examples directory.](/examples/)
+An example of this file structure can be found in the [examples directory.](https://github.com/wrld3d/wrld-indoor-maps-api/tree/master/examples)
 
-##### Main JSON file
+### Main JSON file
 
 The main JSON file is named `main.json` sits in the root of the submitted archive, with the GeoJSON files for each level alongside it.  It contains JSON for a Building object, made up of several levels, described below.
```
Comment (iainbethune) [inline FORMAT.md:43]: "The main JSON file is named `main.json` and sits..."

File: FORMAT.md [hunk 2]
```diff
@@ -108,7 +111,7 @@ In addition to the default GeoJSON members, each feature may specify some additi
 |`type`|string|the type of the feature - this should match one of the strings in “Feature Types,” below.
 |`highlight`|boolean|whether to generate a highlight for the feature. Optional. Defaults to false.
 
-###### Attributes for placeholders 
+#### Attributes for placeholders
```
Comment (iainbethune) [inline FORMAT.md:114]: This section needs to be removed.  Placeholders were an early (abortive) attempt at the functionality now provided by the asset tool / entity sets.  NB.  There must be some support in eegeo-mobile for rendering props at particular scales, it seems 🧐

File: FORMAT.md [hunk 3]
```diff
@@ -206,7 +211,19 @@ A LineString feature representing a path may have some additional properties:
 |`name`|string|The feature's name, for example, `main corridor`. Optional, can be null.
 |`type`|string|The type of the feature. This should match one of the strings in “LineString Feature Types” above.
 
-###### Level path JSON file
+### Entrances
+
+Entrances are used for connecting the internal paths with the external navigation graph.  For our [routing service](https://github.com/wrld3d/wrld-routing-api) it's enough to just define the entrance. For our (Unsupported) Valhalla integration additional steps are required:
```
Comment (iainbethune) [inline FORMAT.md:216]: "unsupported" lower case

File: FORMAT.md [hunk 4]
```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
 ```
 ---
 
-#### Disclaimer
+## Indoor Assets
+
+Indoor Assets are optional pieces of furniture you can add to an Indoor Map.  These are a separate GeoJSON format from the main Indoor Map format that defines all the Indoor Assets for a given floor, which can then be imported using the Import Assets function of the WRLD Indoor Map Designer.
```
Comment (iainbethune) [inline FORMAT.md:305]: "There is a"...

File: FORMAT.md [hunk 5]
```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
...
+
+Each assets GeoJSON file is expected to contain a FeatureCollection in the WGS84 (EPSG: 4326) Coordinate Reference System, and include the appropriate crs attribute. Assets are represented as GeoJSON features with Polygon or Point geometry, with the center of the Polygon or the Point position itself being used as the central point of the Asset.
+
+### Asset Properties
+
+| Attribute| Type  | Description                                                                          |
+| -------- | ----- | ------------------------------------------------------------------------------------ |
+|`id`      |integer|An identifier for the feature. Each asset must have a unique id.
```
Comment (iainbethune) [inline FORMAT.md:313]: Remove the id attribute as it's ignored by the import process (see https://eegeo-team.atlassian.net/browse/MPLY-11811)

File: FORMAT.md [hunk 6]
```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
...
+|`name`    |string |The asset's name, for example, `Desk 42`. Optional, can be null.
+|`type`    |string |Must be set to "InstancedEntity" to be recognized by the Asset Import process.
+|`instance`|string |A Prop ID matching one from our [Prop List](prop-list.html).
```
Comment (iainbethune) [inline FORMAT.md:316]: Prop List link should go to the prop-manifest file on the CDN, I think (until we have the prop-models repo accessible).

File: FORMAT.md [hunk 7]
```diff
@@ -283,13 +300,60 @@ This is represented by a path between the same [long, lat] point on different le
...
+### Example -- desks_geojson_trace.json
+
+```json
+{
+    "type": "FeatureCollection",
+    "features": [
+        { 
+            "type": "Feature", 
+            "properties": {
+                "id": 11,
```
Comment (iainbethune) [inline FORMAT.md:330]: Remove the id property (also from the example files)

File: FORMAT.md (hunk 6 above)
Comment (iainbethune) [reply FORMAT.md:316]: https://cdn-resources.wrld3d.com/props/latest/Assets/manifest.json

File: README.md [hunk 8]
```diff
@@ -154,18 +156,135 @@ curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1/poi/UUID?token=dev_auth_tok
...
+
+### Download an Indoor Asset Set as a GeoJSON
+
+```sh
+curl -v https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/entities.geojson?token=<dev_auth_token> > ./entities.geojson
+```
+
+### Importing Indoor Asset Sets
+
+For more information on how to prepare an Indoor Asset AutoCAD DXF submission, refer to the [Indoor Assets Tutorial](TUTORIAL-ASSETS.md#preparing-an-asset-autocad-dxf-submission).
```
Comment (iainbethune) [inline README.md:228]: links to subsections didn't appear to work for me, please check.

File: README.md [hunk 9]
```diff
@@ -154,18 +156,135 @@ curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1/poi/UUID?token=dev_auth_tok
...
+curl -v -X DELETE https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>?token=<dev_auth_token>
+```
+
+### Publish an Indoor Asset Set
+
+```sh
+curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/publish?token=<dev_auth_token>
+```
+
+For more information on publishing an Indoor Asset Set, see the [Indoor Assets Tutorial](TUTORIAL-ASSETS.md#publishing-your-assets).
```
Comment (iainbethune) [inline README.md:218]: links to subsections didn't appear to work for me, please check.

File: README.md [hunk 10]
```diff
@@ -154,18 +156,135 @@ curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1/poi/UUID?token=dev_auth_tok
...
+
+### Download an Indoor Asset Set as a GeoJSON
+
+```sh
+curl -v https://indoor-maps-api.wrld3d.com/v1.1/indoor-maps/<indoor_uuid>/<floor_id>/entity-sets/<set_id>/entities.geojson?token=<dev_auth_token> > ./entities.geojson
+```
+
+### Importing Indoor Asset Sets
```
Comment (iainbethune) [inline README.md:226]: I think this section needs to be one level up from the Import from DXF and fetching status API examples.

File: README.md [hunk 11]
```diff
@@ -154,18 +156,135 @@ curl -v -XPOST https://indoor-maps-api.wrld3d.com/v1/poi/UUID?token=dev_auth_tok
...
+
```
Comment (iainbethune) [inline README.md:227]: Think we should also add the API for importing a geojson.

File: TROUBLESHOOTING.md [hunk 12]
```diff
@@ -3,26 +3,74 @@ Troubleshooting
...
+The submission service will email you if your map doesn't meet the [specification for our GeoJSON](FORMAT.md).
 
 Common problems are:
+
 1. Incorrect CRS. The CRS (Coordinate Reference System) must be WGS84 ([EPSG: 3857](http://spatialreference.org/ref/sr-org/6864/)).  This can be set at the time of export from QGIS.
-1. Missing ids.  The ids for features can be generated by following the instructions in [the tutorial](TUTORIAL.md#user-content-generate-feature-ids).
+2. Missing ids.  The ids for features can be generated by following the instructions in [the tutorial](TUTORIAL.md#user-content-generate-feature-ids).
+
+
+### Multipolygon errors
```
Comment (iainbethune) [inline TROUBLESHOOTING.md:27]: I didn't realise this section existed here!  Please factor out the content from the bottom of TUTORIAL.md and TUTORIAL-TOOL.md (which is updated by my PR yesterday, so should replace the content here) and replace with a link to this section.

File: TUTORIAL-TOOL.md [hunk 13]
```diff
@@ -334,8 +343,8 @@ $ zip -r my-indoor-map.zip .
 
 Troubleshooting
```
Comment (iainbethune) [inline TUTORIAL-TOOL.md:344]: As per comment in README.md this section can be factored out and replaced by a link to the general troubleshooting section.

File: TUTORIAL.md [hunk 14]
```diff
@@ -357,7 +357,7 @@ If you have any problems feel free to [raise an issue](https://github.com/wrld3d
   Troubleshooting
 ===================
 #### <a name="multipolygons"/>Multipolygons
-If you are receiving an error related to multipolygons in your submission, first check that you ran the [Multipart to Singleparts](#export-level-to-geojson) conversion step when you exported your layer to geojson.
+If you are receiving an error related to multipolygons in your submission, first check that you ran the [Multipart to Singleparts](#export-level-to-geojson) conversion step when you exported your layer to GeoJSON.
```
Comment (iainbethune) [inline TUTORIAL.md:360]: As per comment in README.md this section can be factored out and replaced by a link to the general troubleshooting section.

File: examples/cad-processing/desks_geojson_trace.geojson [hunk 15]
```diff
@@ -0,0 +1,11 @@
+{
+"type": "FeatureCollection",
+"name": "desks_geojson_trace",
+"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
+"features": [
+{ "type": "Feature", "properties": { "id": 11, "type": "InstancedEntity", "name": "MyDesk1", "instance": "white_desk_01", "heading": 70, "height": 0 }, "geometry": { "type": "Polygon", "coordinates": [ [ [ -2.979148078887524, 56.462091078201396 ], [ -2.979126744285509, 56.462095184442788 ], [ -2.979122727310234, 56.462088578750119 ], [ -2.979144151178366, 56.462083936912023 ], [ -2.979148078887524, 56.462091078201396 ] ] ] } },
```
Comment (iainbethune) [inline examples/cad-processing/desks_geojson_trace.geojson:6]: Remove IDs

File: examples/cad-processing/floorplan/main.json [hunk 16]
```diff
@@ -0,0 +1,18 @@
+{
+	"id": "EIM-0e474625-21bc-4ba1-872e-c4fffebb798a",
+	"name": "Example office",
+	"owner": "eeGeo",
```
Comment (iainbethune) [inline examples/cad-processing/floorplan/main.json:4]: WRLD

File: README.md (hunk 8 above)
Comment (SirPryderi) [reply README.md:228]: The link works correctly in the readme page, the images are loading late, tho. This caused the text to be pushed down, after the browser was done scrolling.

File: README.md (hunk 10 above)
Comment (SirPryderi) [reply README.md:226]: I pushed the subsections one level instead, so it's still under `indoor assets`.
image

File: README.md (hunk 11 above)
Comment (SirPryderi) [reply README.md:227]: Took me a while to **not** find it. There's no API for that, the geojson gets parsed by the JavaScript client and then the bulk update endpoint is used.

File: README.md (hunk 9 above)
Comment (SirPryderi) [reply README.md:218]: The link works correctly in the readme page, the images are loading late, tho. This caused the text to be pushed down, after the browser was done scrolling.

**Pull request merged**